### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by YektaDev** [gurland/telegram-delete-all-messages#20]
Tested on Windows 10 CMD, Log:

```
> install.bat

python -m venv venv
call venv/Scripts/activate.bat
Requirement already satisfied: pyrogram in c:\...\telegram-delete-all-messages-master\venv\lib\site-packages (from -r requirements.txt (line 1)) (1.0.7)
Requirement already satisfied: tgcrypto in c:\...\telegram-delete-all-messages-master\venv\lib\site-packages (from -r requirements.txt (line 2)) (1.2.2)
Requirement already satisfied: async-lru==1.0.2 in c:\...\telegram-delete-all-messages-master\venv\lib\site-packages (from pyrogram->-r requirements.txt (line 1)) (1.0.2)
Requirement already satisfied: pyaes==1.6.1 in c:\...\telegram-delete-all-messages-master\venv\lib\site-packages (from pyrogram->-r requirements.txt (line 1)) (1.6.1)
Requirement already satisfied: pysocks==1.7.1 in c:\...\telegram-delete-all-messages-master\venv\lib\site-packages (from pyrogram->-r requirements.txt (line 1)) (1.7.1)
"Successfully installed requirements!"
```

```
> start.bat

call venv/Scripts/activate.bat
Traceback (most recent call last):
  File "cleaner.py", line 5, in <module>
    from pyrogram.api.functions.messages import Search
ModuleNotFoundError: No module named 'pyrogram.api'
```

**Comment by SoHardDay** [comment]
https://github.com/gurland/telegram-delete-all-messages/pull/17

**Comment by gurland** [comment]
Thanks for your report! Fixed in #17 PR.
